src/components/Router/StepComponentMap.ts

```diff
@@ -265,9 +265,14 @@ const buildDocumentComponents = (
     countryCode,
     documentType
   )
+  const hasMultipleDocumentsWithUnsupportedCountry =
+    !hasOnePreselectedDocument && !supportedCountry
+  const hasCountryCodeNullOrBooleanDocumentTypeFlag =
+    countryCode !== null || configForDocumentType === true
```

Comment:
Why not 
```suggestion
    countryCode !== null || configForDocumentType
```

Reply:
Actually, realised this would need to be a strict check for a boolean `true` value since `configForDocumentType` could also be an object which is truthy also.